contrib/syslog-ng-debun

```diff
@@ -857,6 +857,39 @@ acquire_syslog_nprv () {
 	acquire_syslog_startup_method
 }
 
+remove_ssl_passwords_from_file () {
+	#
+	# Double quotes are used because escaping single quotes within a single quoted string is ugly.
+	# [ \t\r\n\v\f] class is used for matching whitespaces, because \s is unavailable.
+	# The goal is matching substrings that look like this, to replace the password string, and preserve original formatting:
+	# ssl.keystore.password ( 'password-string' ) -->  ssl.keystore.password ( '___PASSWORD_REMOVED___' )
+	# ssl.key.password ( 'password-string' ) -->  ssl.key.password ( '___PASSWORD_REMOVED___' )
+	# ssl.truststore.password ( 'password-string' ) -->  ssl.truststore.password ( '___PASSWORD_REMOVED___' )
+	###
+	
+	${sed_equivalent_cmd} "s|ssl.keystore.password([ \t\r\n\v\f]*)\(([ \t\r\n\v\f]*)([\"']?)[^)\"']+([\"']?)([ \t\r\n\v\f]*)\)|ssl.keystore.password\1(\2\3___PASSWORD_REMOVED___\4\5)|g" <"${1}" >"${1}.bak1" 2>/dev/null
+	${sed_equivalent_cmd} "s|ssl.key.password([ \t\r\n\v\f]*)\(([ \t\r\n\v\f]*)([\"']?)[^)\"']+([\"']?)([ \t\r\n\v\f]*)\)|ssl.key.password\1(\2\3___PASSWORD_REMOVED___\4\5)|g" <"${1}.bak1" >"${1}.bak2" 2>/dev/null
+	${sed_equivalent_cmd} "s|ssl.truststore.password([ \t\r\n\v\f]*)\(([ \t\r\n\v\f]*)([\"']?)[^)\"']+([\"']?)([ \t\r\n\v\f]*)\)|ssl.truststore.password\1(\2\3___PASSWORD_REMOVED___\4\5)|g" <"${1}.bak2" >"${1}.bak3" 2>/dev/null
+	mv "${1}.bak3" "${1}"	
+	rm -f "${1}".bak*
+}
+
+acquire_kafka_properties_file () {
+	
+	if [ -f "${tmpdir}/syslog-ng.ctl.running.conf" ]; then
+		prop_file=$( sed -n "/kafka/,/);/ p" ${tmpdir}/syslog-ng.ctl.running.conf | grep properties-file | sed 's/.*(\(.*\))/\1/' | sed 's/"//g' )
```

Comment:
I would recommend using ${sed_equivalent_cmd} instead of sed, but grep will likely be okay.